src/charselect.cpp

```diff
@@ -168,7 +168,7 @@ void Courtroom::char_clicked(int n_char)
     }
   }
 
-  if (n_char != m_cid) {
+  if (n_char != m_cid || n_char == -1) {
```

Comment:
I don't believe this will work on all servers because not all servers recognize CID -1, or if they do they won't answer with anything.